packages/dx-react-scheduler/docs/guides/grouping.md

```diff
@@ -11,16 +11,13 @@ Appointments can be grouped by [resources](../reference/resources.md). Group nam
 
 ## Basic Usage
 
-To group by resources, [configure them](./resources.md) first.
-
-Grouping is implemented in the [IntegratedGrouping](../reference/integrated-grouping.md) plugin. Add this and the [GroupingState](../reference/grouping-state.md) plugin to enable this feature.
+To group by resources, [configure them](./resources.md) first. Then, add the [IntegratedGrouping](../reference/integrated-grouping.md) and [GroupingState](../reference/grouping-state.md) plugins.
```

Comment:
It would be better. Because Integrated plugin requires State plugin

```suggestion
To group by resources, [configure them](./resources.md) first. Then, add the [GroupingState](../reference/grouping-state.md) and [IntegratedGrouping](../reference/integrated-grouping.md) plugins.
```